src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingSpec.cs

```diff
@@ -58,7 +58,9 @@ public class ClusterShardingSpecConfig : MultiNodeClusterShardingConfig
             CommonConfig = ConfigurationFactory.ParseString($@"
                 akka.cluster.sharding.verbose-debug-logging = on
                 #akka.loggers = [""akka.testkit.SilenceAllTestEventListener""]
-
+                akka.cluster.downing-provider-class = ""Akka.Cluster.SBR.SplitBrainResolverProvider, Akka.Cluster""
```

Comment:
This spec previously relied on `auto-down-unreachable-after` - in order to make it pass again we needed to:

* Enable the SBR
* Set the `akka.cluster.split-brain-resolver.stable-after` to the lowest possible value (must be > 0.0s)
* Set the `akka.cluster.down-removal-margin` to the lowest possible value (must be > 0.0s)